src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -175,7 +181,7 @@ public partial class FluentDataGrid<TGridItem> : FluentComponentBase, IHandleEve
     /// <summary>
     /// Gets the first (optional) SelectColumn
     /// </summary>
-    internal IEnumerable<SelectColumn<TGridItem>> SelectColumns => _columns.Where(col => col is SelectColumn<TGridItem>).Cast< SelectColumn<TGridItem>>();
+    internal IEnumerable<SelectColumn<TGridItem>> SelectColumns => Columns.Where(col => col is SelectColumn<TGridItem>).Cast< SelectColumn<TGridItem>>();
```

Comment:
why accept multiple selection columns? 
I will raise an exception,

Reply:
never seen other grids with multiple selection columns

Reply:
It will not be _the_ most used function, but let's not rule it out. Think about maybe switching out a multi select column for a single select one, showing and hiding one or the other based on some condition or user interaction.